docs/basics/app-anatomy.md

```diff
@@ -139,7 +139,9 @@ Modules must implement [interfaces](../building-modules/module-manager.md#applic
 
 ### `Msg` Services
 
-Each module defines two [Protobuf services](https://developers.google.com/protocol-buffers/docs/proto#services): one `Msg` service to handle messages, and one gRPC `Query` service to handle queries. If we consider the module as a state-machine, then a `Msg` is a state transition. A `Msg` service is a Protobuf service defining all possible `Msg`s a module exposes. Note that `Msg`s are bundled in [`transactions`](../core/transactions.md), and each transaction contains one or multiple `messages`.
+Each module defines two [Protobuf services](https://developers.google.com/protocol-buffers/docs/proto#services): one `Msg` service to handle messages, and one gRPC `Query` service to handle queries. If we consider the module as a state-machine, then a `Msg` service is a set of state transition RPC methods.
+Each Protobuf `Msg` service RPC is 1:1 related to a Protobuf request type, which must implement `sdk.Msg` interface.
```

Comment:
So in the [protobuf docs](https://developers.google.com/protocol-buffers/docs/proto#services), a service == a RPC service, and that's where RPC is mentioned. What we call "RPC" here is actually what protobuf calls a method.

My proposal would be to `s/service RPC/service method/g` (in all docs), what do you think?

Reply:
I was thinking about it, I thought that we use `method` because of Go, and in proto each endpoint is marked as RPC. But if in prodobuf docs they use _method_ then let's use it here as well. I'm updating the docs.